tests/components/Dropdown.test.tsx

```diff
@@ -101,19 +102,19 @@ describe('Dropdown', () => {
       </Dropdown>
     );
     const inputNode = screen.getByRole('textbox');
-    userEvent.click(inputNode);
+    await userEvent.click(inputNode);
     const itemNode = screen.getByText('item1');
 
-    userEvent.keyboard('{Tab}');
+    await userEvent.keyboard('{Tab}');
     expect(itemNode.className).toContain('FocusedItem1');
     expect(inputNode).toHaveValue('item1');
 
-    userEvent.keyboard('{Shift}{Tab}');
+    await userEvent.keyboard('{Shift>}{Tab}{/Shift}');
```

Comment:
why is this change needed?

Reply:
the keyboard wasn't triggering properly so i switched to this format